repository/agenda-general.js

```diff
@@ -123,7 +123,14 @@ const selectAgendaitemsForSorting = async (agendaURI) => {
       ${sparqlEscapeUri(agendaURI)} dct:hasPart ?agendaitem .
       ?agendaitem a besluit:Agendapunt ;
         schema:position ?number ;
-        ext:wordtGetoondAlsMededeling ?isRemark .
+        dct:type ?type .
+      BIND(
+        IF (
+          ?type = <http://themis.vlaanderen.be/id/concept/agendapunt-type/8f8adcf0-58ef-4edc-9e36-0c9095fd76b0>,
```

Comment:
URI specifieren in een `const` variable zoals de andere URIs in deze service ipv hardcoded in query